composer.json

```diff
@@ -54,7 +55,10 @@
         "laravel": {
             "providers": [
                 "Rapidez\\Statamic\\RapidezStatamicServiceProvider"
-            ]
+            ],
+            "aliases": {
+                "StatamicNav": "Rapidez\\Statamic\\Facades\\StatamicNav"
```

Comment:
Do we want a facade this specific for `StatamicNav`? 
Maybe a facade for e.g. `RapidezStatamic` so any future functionalities can be added without having to create separate facades and possibly moving `StatamicNav` to that as well